app/src/lib/git/rev-list.ts

```diff
@@ -119,13 +119,25 @@ export async function getBranchAheadBehind(
  * Returns `null` when the rebase is not possible to perform, because of a
  * missing commit ID
  */
-export async function getCommitsInRange(
+export async function getCommitsBetweenCommits(
```

Comment:
Given there are two methods to retrieve commits, maybe it makes sense to add a note to the comment above to specify that `getCommitsBetweenCommits` doesn't include the "from" commit. What do you think?

Reply:
This is the between commits  comment
>* Get a list of commits from the target branch that do not exist on the base
>* branch, ordered how they will be applied to the base branch. `

The "from" parameter is the `baseBranchSha`. The comment does seem to cover that it only includes commits on the target branch that don't exist on the base branch. This is the same as saying it would not have the commit referenced by the `baseBranchSha` as it would be on the base branch.

Reply:
Add an line in addition 
> Therefore, this will not include the baseBranchSha commit.

To hopefully be more clear.